clients/nodejs/nimiq

```diff
@@ -8,9 +8,9 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
 elif [[ "$OSTYPE" == "darwin"* ]]; then
     MB_MEMORY=$(sysctl hw.memsize | awk '{mb =int($2/1024/1024); print mb}')
 else
-    node "$SCRIPT_PATH/index.js" "$@"
+    NIMIQ_WS_ENGINE="uws" node "$SCRIPT_PATH/index.js" "$@"
```

Comment:
Hmm, good point. The reason I changed it here too is because I wasn't sure if the previous matches would cover 100% of Linux and Mac platforms (also it didn't occur to me that someone would try to run this on cygwin or similar :joy: ), but I think it makes sense to remove it as it only covers a very unlikely edge-case.